package.json

```diff
@@ -1314,13 +1319,13 @@
               "type": "coreclr",
               "request": "launch",
               "preLaunchTask": "build",
-              "program": "^\"\\${workspaceRoot}/bin/Debug/${1:<target-framework>}/${2:<project-name.dll>}\"",
```

Comment:
All of this part of launch.json is generated from src/tools/OptionsSchema.json. Can you fix it there too? Otherwise this will be lost the next time we run the generator.

Reply:
Looks like this is fixed now in the latest master.